airflow/contrib/kubernetes/kubernetes_request_factory/kubernetes_request_factory.py

```diff
@@ -191,3 +191,10 @@ def extract_image_pull_secrets(pod, req):
     def extract_tolerations(pod, req):
         if pod.tolerations:
             req['spec']['tolerations'] = pod.tolerations
+
+    @staticmethod
+    def extract_env_from(pod, req):
+        for idx, configmap in enumerate(pod.envs_from_configmaps):
+            req['spec']['containers'][0]['envFrom'][idx]['configMapRef']['name'] = configmap.name
+        for idx, secret in enumerate(pod.envs_from_secrets):
+            req['spec']['containers'][0]['envFrom'][idx]['secretRef']['name'] = secret.name
```

Comment:
There is already support for `secretRef` in  `extract_env_and_secrets` -  please add config maps to there rather than creating a new function.

Reply:
@ashb done, fixed.